### PR TITLE
[New-Rule] Suspicious WMI Image Load from MS Office

### PR DESCRIPTION
**Pull request by dstepanic17**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/543

## Summary
Detects suspicious activity related to WMI image load (wmiutils.dll) from MS Office products

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

## Example Data
image

### CODE REVIEW

File: rules/windows/execution_suspicious_image_load_wmi_ms_office.toml [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+ecs_version = ["1.6.0"]
+maturity = "production"
+updated_date = "2020/11/17"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies suspicious image load (wmiutils.dll) from Microsoft Office processes. This behavior may indicate adversarial
+activity where child processes are spawned via Windows Management Instrumentation (WMI), this technique is used to
+execute code and evade traditional parent/child processes spawned from MS Office products.
```
Comment (bm11100) [inline rules/windows/execution_suspicious_image_load_wmi_ms_office.toml:12]:
```suggestion
Identifies a suspicious image load (wmiutils.dll) from Microsoft Office processes. This behavior may indicate adversarial
activity where child processes are spawned via Windows Management Instrumentation (WMI). This technique can be used to
execute code and evade traditional parent/child processes spawned from MS Office products.
```

Comment (bm11100) [reply rules/windows/execution_suspicious_image_load_wmi_ms_office.toml:12]: may need to re-lint afterwards

**Review by Samirbous — APPROVED** [review]

File: rules/windows/execution_suspicious_image_load_wmi_ms_office.toml [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+Identifies a suspicious image load (wmiutils.dll) from Microsoft Office processes. This behavior may indicate
+adversarial activity where child processes are spawned via Windows Management Instrumentation (WMI). This technique can
+be used to execute code and evade traditional parent/child processes spawned from MS Office products.
+"""
+index = ["winlogbeat-*", "logs-endpoint.events.*"]
+language = "eql"
+license = "Elastic License"
+name = "Suspicious WMI Image Load from MS Office"
+references = [
+    "https://medium.com/threatpunter/detecting-adversary-tradecraft-with-image-load-event-logging-and-eql-8de93338c16",
```
Comment (bm11100) [inline rules/windows/execution_suspicious_image_load_wmi_ms_office.toml:19]: great blog

**Review by bm11100 — APPROVED** [review]
🧯

**Pull request merged**